backend/.env.example

```
@@ -3,6 +3,4 @@ POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
 POSTGRES_DB=jamplay
 POSTGRES_PORT=5432
-DB_APP_USER=jamplay_app
-DB_APP_PASSWORD=jamplay1234
-DATABASE_URL="postgresql://jamplay_app:jamplay1234@localhost:5432/jamplay?schema=public"
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/jamplay?schema=public"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the POSTGRES_DB key

(UnorderedKey)